### DISCUSSION TITLE
Transition to tabs

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#779] [Ideas]
### What feature would you like to see?

I suggest adding a subtle transition to the tabs when clicking on them considering they increase their size, even if it's for less than a second I would look nicer, and better.

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/638_

**Comment by andermendz** [comment]
not a feature but just a little tweak, the workspace element is not distanced the same as the pages : 
![image](https://github.com/user-attachments/assets/af989136-2616-43cc-af1f-55d930449646)
